config/defaults.mk

```diff
@@ -27,7 +27,7 @@ DEBUG_FLAGS = -g $(MFEM_XCOMPILER) -Wall $(BASE_FLAGS)
 
 # Destination location of make install
 # PREFIX = $(HOME)/mfem
-PREFIX = ./mfem
+PREFIX = .
```

Comment:
@v-dobrev Changed this to the current directory since that's the default installation path

Reply:
Nevermind, @tzanio mentioned why it's `./mfem`. Reverting this change